src/Common/ProgressIndication.cpp

```diff
@@ -190,9 +190,11 @@ void ProgressIndication::writeProgress()
     if (cpu_usage > 0 || memory_usage > 0)
     {
         WriteBufferFromOwnString profiling_msg_builder;
-
-        profiling_msg_builder << "(" << fmt::format("{:.1f}", cpu_usage) << " CPU";
-
+        profiling_msg_builder << "(";
+        if (cpu_usage <= 0)
+            profiling_msg_builder << 0 << " CPU";
```

Comment:
It will introduce unwanted jumping effect when two messages

0 CPU
0.1 CPU

will quickly change each other.

It is better to have

0.0 CPU
0.1 CPU

for alignment.